.editorconfig

```diff
@@ -0,0 +1,17 @@
+# editorconfig.org
+
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
```

Comment:
Exactly.. I thought so.. and 2 because here there were 2: https://github.com/LeaVerou/bliss/blob/gh-pages/package.json and since you are using tab.. it doesn't matter :/